sci-biology/cufflinks/files/cufflinks-2.2.1-gcc6.patch

```diff
@@ -0,0 +1,14 @@
+--- a/src/lemon/error.h
++++ b/src/lemon/error.h
```

Comment:
Maybe use the debian patch instead: https://sources.debian.net/src/cufflinks/2.2.1-3/debian/patches/fix_gcc-6.1_issue_in_liblemon_code_copy.patch/
Anyhow, cufflinks is awful C++, and either way both ways are the wrong way to handle errors (it should throw an exception really). If you integrate the debian patch, we can merge it anyways, though.